src/components/ui/theme-toggle.tsx

```
@@ -0,0 +1,39 @@
+import { Moon, Sun } from "lucide-react";
+import { useEffect, useState } from "react";
+
+import { Button } from "@/components/ui/button";
+
+export function ThemeToggle() {
+	const [theme, setTheme] = useState<"light" | "dark">("dark");
+
+	useEffect(() => {
```

Biome (2.1.2)

[error] This hook does not specify its dependency on applyTheme.

This dependency is being used here, but is not specified in the hook dependency list.

Either include it or remove the dependency array.
Unsafe fix: Add the missing dependency to the list.


(lint/correctness/useExhaustiveDependencies)